src/components/CreateCard/Cards.js

```diff
@@ -0,0 +1,162 @@
+import React, {useEffect, useState} from "react";
+import {axiosWithAuth} from "../../utils/axios";
+import {Card, Button, Icon, Popover, Modal, Input, Select, Avatar, Spin} from "antd";
+import {Link} from "react-router-dom";
+import {getCards, deleteCard, getDeckId} from "../../state/actions/CardAction";
+import {connect} from "react-redux";
+import styles from "./AddCard.module.css";
+import HeaderSearchBar from "../ListDeckInfo/HeaderSearchBar";
+import Dashboard from "../../layout/Dashboard/Dashboard";
+import AddCard from "./AddCard";
+import {EditOutlined, DeleteOutlined} from "@ant-design/icons";
```

Comment:
Why are you using a separate Icon package? antd has all the same things by default.